docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
I doubt I could come up with anything you haven't already considered. Some generic terms I've seen in descriptions of message-passing systems are `channel`, `messagebroker` or just `broker`, `center` (as in `NSNotificationCenter`), or just the good ol' `publisher`. Not sure I like any of those better than `daemon`, personally.
`</bikeshed>`